saleor/graphql/attribute/utils.py

```diff
@@ -161,6 +180,24 @@ def _get_assigned_attribute_value_if_exists(
             else assignment.values.filter(**{lookup_field: value}).first()
         )
 
+    @classmethod
+    def _create_value_instance(cls, attribute, attr_value, external_ref):
+        existing_slugs = get_existing_slugs(attribute, [attr_value])
```

Comment:
IMO, we should refactor it somehow. This method is fetching the existing slugs, and we are calling this method on each value. We should fetch the slug values earlier and update them every time a new slug is created.